src/app/layouts/navbar.component.html

```diff
@@ -24,6 +24,11 @@
         <li class="nav-item" routerLinkActive="active">
           <a class="nav-link" routerLink="/tools">{{ "tools" | i18n }}</a>
         </li>
+        <ng-container *ngIf="showOrganizations">
+          <li class="nav-item" routerLinkActive="active">
+            <a class="nav-link" routerLink="/organizations">{{ "organizations" | i18n }}</a>
```

Comment:
I've been thinking on this a bit, we could just link directly to the first organization if they only have a single organization. Similar to how the provider link works. Which would let us get rid of the router guard. No real preference though, but would be nice to update the provider to use the same pattern if we switch?

Reply:
The nice thing about the router guard is that it lets the navbar direct to a route without having to know where it goes. But, it does duplicate this logic a little, and it's a bit counterintuitive having a `canActivate` guard which always returns false. I'll bring it in line with the providers.

Reply:
This is done